public/static/app/waterfall/waterfall.jsx

```diff
@@ -12,6 +12,10 @@ function getFormattedTime(input, userTz, fmt) {
   return moment(input).tz(userTz).format(fmt);
 }
 
+function formatMessageWithGitTags(message, git_tags) {
```

Comment:
Could we have the git tags be on a separate line from the commit message? My thinking is that if the git tags are an important piece of info, putting it at the end of the commit message will make it get lost, whereas having it be on its own line will call it out a bit more

Reply:
I was also thinking this but I mentioned in the description that I couldn't figure out how to separate message and git tags since new line things in js is being ignored for some reason. I can play around with it some more though! 

Do you think it's okay to have git tags still be in the collapsable message (i.e. only visible if you expand with "more") or should it be separate from that even? I was worried about putting too much in the header.